code/datums/supplypacks.dm

```diff
@@ -733,6 +786,45 @@ var/list/all_supply_groups = list("Supplies","Clothing","Security","Hospitality"
 	containername = "plasmaman suit crate"
 	group = "Clothing"
 
+/datum/supply_packs/amogus
+	name = "Theatric Space Suits"
```

Comment:
I don't get what makes these theatrical. These are just "order as many 4TC syndicate space suits as you want", aren't they?

These enjoy a substantial boost over the normal space suits.
+60 melee, +50 bullet, +30 laser, +15 energy, +30 bomb, better storage slot, fits inside a standard backpack -- these things are great and worth 4TC. But now you can get all you want because you bought an emag?

Too much emag creep!

Reply:
look at the name a little closer, where else do you get space suits that look so similar but just have different colors?

Reply:
I would rather see a wardrobe that just gives you a bunch of fake suits and you get all of them instead of an "order one random uplink item suit at a time". It suits your gimmick way better and isn't nearly so creepy

Reply:
Make the bike horns use the amogus sound effect if you're wearing those suits.